src/components/TextInput.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Field } from 'redux-form';
+import { TextField } from 'redux-form-material-ui';
+import PropTypes from 'prop-types';
+
+
+const TextInput = ({ name, placeholder, ...props }) => (
+  <Field
+    name={name}
+    component={TextField}
+    {...props}
+    placeholder={placeholder}
```

Comment:
I looked up this attribute, it's not part of redux-forms `Field`, so it will be passed to `TextField` from `material-ui`.

So it has no special meaning here, it might as well be passed through `...props` and then we can drop this code.